scripts/bootstrap.sh

```diff
@@ -1,11 +1,25 @@
 #!/bin/bash
 set -euo pipefail
 
+if [ -z "$APP_COMPONENT" ]; then
+  echo "Please set APP_COMPONENT"
+  exit 1
+fi
+
 echo "Waiting for SQL to become available..."
 
 ./scripts/await_service.sh $SQL_HOST $SQL_PORT $SERVICE_READINESS_TIMEOUT
 
 ./scripts/ensure_sql.sh
 ./scripts/ensure_meili.sh
 ./scripts/migrate.sh up
-exec ./scripts/run.sh
+
+echo $APP_COMPONENT
```

Comment:
guessing this is a debug log

Reply:
bash flag `set -x` may be of interest to you for debugging

(can be dangerous as it can print secrets so i typically only use euo in real envs)